docs/source/contribution.rst

```diff
@@ -202,15 +69,15 @@ Coding Guidelines
 
 .. note::
 
-   Coding guidelines are updated at v3.0.
-   Those who have contributed to older versions should read the guidelines again.
+   Coding guidelines are updated in v3.0.
+   Those who have contributed to older versions may want to read the updated guidelines.
```

Comment:
Can we remove this note because v3 is anyway very old?

Reply:
I agree, removed the note.